schema/project-level/project-schema.json

```
@@ -297,6 +297,21 @@
       "default": "en",
       "minLength": 1
     },
+    "lobbyingMeetings": {
+      "title": "Lobbying meetings",
+      "description": "Information about lobbying meetings in relation to the project. That is, meetings with public officials that are made, managed or directed with the purpose of influencing public decision-making in relation to the project, whether for private, public or collective ends, whether for compensation or without. Public decision-making is the creation and amendment of legislation or any other regulatory measures; the development, modification and implementation of public policies, strategies and programmes; and the awarding of government contracts or grants, administrative decisions or any other public spending decisions.",
+      "type": "array",
+      "items": {
+        "$ref": "#/definitions/Meeting"
+      },
+      "uniqueItems": true,
+      "minItems": 1
+    },
+    "social": {
+      "title": "Social",
+      "description": "Information about the social impact of the project and actions the project will take to mitigate potentially negative impacts on adjacent populations and project workers.",
```

To align with changes to `environment` in another PR.

Repeat the changes for the Social definition.


```suggestion
      "description": "Information about the social impacts of the project and the actions the project will take to mitigate potentially negative impacts on adjacent populations and project workers.",
```